manifests/supervisorcluster/1.25/cns-csi.yaml

```diff
@@ -222,6 +222,7 @@ spec:
             - "--timeout=300s"
             - "--csi-address=$(ADDRESS)"
             - "--feature-gates=Topology=true"
+            - "--extra-create-metadata=true"
```

Comment:
we need to make sure following arg's only gets added when we have pod vm on stretched supervisor cluster fss enabled on the vc.

```
  "--podvm-on-stretched-supervisor=true",
   "--extra-create-metadata=true"
```

Reply:
This requires additional changes on customized csi provisioner for wcp.
@xing-yang Any comments on this ?

Reply:
I'd prefer not to add customized code changes on external-provisioner.

Reply:
@shalini-b has already added code change for `podvm-on-stretched-supervisor` gate in the gitlab repo.

Reply:
for now can you keep both args as false for provisioner. we will enable them later after testing is complete.

Reply:
Done